jf_agent/main.py

```diff
@@ -191,7 +179,15 @@ def main():
 
         logger.info(f"Mounted old output directory as {config.outdir}, will attempt to send.")
 
-    else:
+    elif not config.run_mode == 'send_only':
+        # Importantly, don't overwrite the already-existing diagnostics file
+        try:
+            full_validate(config, creds, jellyfish_endpoint_info)
```

Comment:
thats a great point. I think what I will do is enable the jira side from jf_ingest (takes ~10-15 mins in the worst case) and skip git temporarily since we are migrating that over to jf_ingest as well. then at a later time I will add the git validation back in via jf_ingest